examples/chants/components/chants/chant.css.js

```diff
@@ -0,0 +1,39 @@
+import css from "https://tfl.dev/@truffle/utils@~0.0.3/css/css.ts";
+
+export default css`
+@keyframes pop {
+  50%  {transform: scale(2);}
```

Comment:
want to make the spacing a little cleaner?
```
50% {
  transform: scale(2);
}
```

(don't think we have a sass linter atm